### DISCUSSION TITLE
Images on the client side

### DISCUSSION THREAD

**Discussion by dgfreecodecamper** [dolanmiu/docx#889] [Q&A]
I'm trying to display images in the docx file which is generated on the client-side. The image is from a URL rather than the local file system. Any examples of how to achieve this? Alternatively, do you have an example of generating the docx file from a lambda function?

**Comment by dolanmiu** [comment]
Handling of images are out of scope of the library

But if you are using Angular for example, you could use the `HttpClient`, then turn it into a base64 string, then add the image as a base64 string.

Lambda function just a nodeJS function on AWS, so it will be the similar as how RunKit does it:

https://runkit.com/dolanmiu/docx-demo1

**Reply by dolanmiu** [reply]
https://stackblitz.com/edit/react-docx-images
